### PR TITLE
Added Advanced Options Panel For Future Extensions (Issue #10)

### PR DESCRIPTION
**Pull request by krooldonutz**
Added Advanced Options Panel For Future Extensions

[Issue 10](https://github.com/smileyy/scalangband/issues/10)

### CODE REVIEW

**Comment by smileyy** [comment]
Having trouble with github with looking at the diff right now, Also, can you open these against branches in the project so it is easier for me to pull down and test out? Or do you not have permission to create branches in the project?

File: main/scala/scalangband/ui/NewGameWindow.scala [hunk 1]
```diff
@@ -7,41 +7,55 @@ import scalangband.model.settings.Settings
 import scalangband.model.{Game, Player}
 
 import scala.swing.event.ButtonClicked
-import scala.swing.*
+import scala.swing._
```
Comment (smileyy) [inline main/scala/scalangband/ui/NewGameWindow.scala:10]: Is there a reason this needed to change to `._`? I think `.*` is the preferred Scala 3 format.

**Comment by krooldonutz** [comment]
yess i don't think i have access to to make a branch from the main repo

File: main/scala/scalangband/ui/NewGameWindow.scala (hunk 1 above)
Comment (krooldonutz) [reply main/scala/scalangband/ui/NewGameWindow.scala:10]: hmm let me check on this after work, i had some problem with intellij and had to mess around with the imports.
let me check later if i'm able to use the imports from the original repo

**Comment by smileyy** [comment]
I'm going to go ahead and pull this in and check it out. The * import is easy enough to fix at some point.

**Pull request merged**